go-controller/pkg/ovn/egressip_test.go

```diff
@@ -833,34 +1272,30 @@ var _ = ginkgo.Describe("OVN master EgressIP Operations", func() {
 				gomega.Expect(egressIPs[0]).To(gomega.Equal(egressIP.String()))
 
 				podUpdate := newPod(namespace, podName, node1Name, podV6IP)
-
+				ginkgo.By("Bringing down NBDB")
+				// inject transient problem, nbdb is down
+				fakeOvn.controller.nbClient.Close()
+				gomega.Eventually(func() bool {
+					return fakeOvn.controller.nbClient.Connected()
+				}).Should(gomega.BeFalse())
 				_, err = fakeOvn.fakeClient.KubeClient.CoreV1().Pods(egressPod.Namespace).Update(context.TODO(), podUpdate, metav1.UpdateOptions{})
 				gomega.Expect(err).ToNot(gomega.HaveOccurred())
+				time.Sleep(types.OVSDBTimeout + time.Second)
+				// check to see if the retry cache has an entry
+				key := getNamespacedName(podUpdate.Namespace, podUpdate.Name)
+				gomega.Eventually(func() *retryObjEntry {
+					return fakeOvn.controller.retryEgressIPPods.getObjRetryEntry(key)
+				}).ShouldNot(gomega.BeNil())
```

Comment:
same thing here, `oldObj` and `newobj` `!= nil`

Reply:
sorry, I pressed enter here before sending the other comments.... I meant "same thing" as: https://github.com/ovn-org/ovn-kubernetes/pull/2965#discussion_r867883623